pkgs/development/libraries/ygor/default.nix

```diff
@@ -0,0 +1,67 @@
+
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, boost
+, gsl
+, eigen
+, gnuplot
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ygor";
+  version = "20200415.1";
+
+  src = fetchFromGitHub {
+    owner = "hdclark";
+    repo = "ygor";
+
+    # Reminder: the sha256 hash can be computed via:
+    #  nix-prefetch-url --unpack "https://github.com/hdclark/ygor/archive/${rev}.tar.gz"
+    #
```

Comment:
Thanks for letting me know. I found it useful to document since a standard `wget '...' -O - | sha256sum -` won't work. However, I'm fine with removing it if necessary.